components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/util/OIDCSessionManagementUtil.java

```diff
@@ -202,6 +221,21 @@ public static Cookie removeOPBrowserStateCookie(HttpServletRequest request, Http
         return null;
     }
 
+    /**
+     * Resolve the user login tenant domain.
+     *
+     * @param request
+     * @return tenantDomain
+     */
+    private static String resolveTenantDomain(HttpServletRequest request) {
+        String tenantDomain = request.getParameter(FrameworkConstants.RequestParams.LOGIN_TENANT_DOMAIN);
```

Comment:
new like after the method signature

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1622/commits/7d8e344901ed2569cd28733bed3f2a770a982271